about.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>AC About</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&family=Roboto:wght@300;900&display=swap" rel="stylesheet">
+  <link
+  rel="stylesheet"
+  href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+  integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+  crossorigin="anonymous"
+  /> 
+  <link rel="stylesheet" href="styles/about.css">
+
+</head>
+<body>
+  <header>
+    <div class="nav-top">
+      <ul class="nav-menu-top ">
+        <li class="nav-item-top"><i class="fab fa-facebook-f nav-link-top"></i></li>
+        <li class="nav-item-top"><i class="fab fa-twitter nav-link-top"></i></li>        
+        <li class="nav-item-top"><a href="#" class="nav-link-top">English</a></li>
+        <li class="nav-item-top"><a href="#" class="nav-link-top">My Page</a></li>
+        <li class="nav-item-top"><a href="#" class="nav-link-top">Login</a></li>
+      </ul>
+    </div>
+
+    <nav class="navbar">
+      <a href="#" class="logo">AC <span>2022</span></a>
+      <ul class="nav-menu ">
+        <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+        <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+        <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+        <li class="nav-item"><a href="#" class="nav-link">AC Campaign</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="hero">
+      <h2>"Hello World! We Are Algerian Coders</h2>
```

Comment:
- [ ] [Optional] Following the Figma design, Kindly hide this in desktop view as it does not show in the design desktop view, you can do that with media query